src/components/Modal/ModalContainer.tsx

```diff
@@ -8,23 +8,41 @@ import { Modal } from './Modal';
 export const ModalContainer: React.FC<ModalContainerProps> = ({
   onClose,
   addTodo,
+  editTodo,
+  currIDForEdit,
+  modalType,
+  initialValueForModalWithEditFeature,
 }) => {
-  const [title, handleTitle] = useInput();
-  const [description, handleDescription] = useInput();
+  const { title: initialTitle } = initialValueForModalWithEditFeature;
+  const { description: initialDescription } =
+    initialValueForModalWithEditFeature;
 
+  const [title, handleTitle] = useInput(
+    modalType === 'edit' ? initialTitle : ''
+  );
+  const [description, handleDescription] = useInput(
+    modalType === 'edit' ? initialDescription : ''
+  );
```

Comment:
When you invoke the creating a new todo- you don't add any initial value, You can use initial values without additional checks. Don't forget to erase the initial value, while closing the window

Reply:
done, true story